src/OrchardCore.Modules/OrchardCore.Deployment/Manifest.cs

```diff
@@ -5,5 +5,6 @@
     Author = ManifestConstants.OrchardCoreTeam,
     Website = ManifestConstants.OrchardCoreWebsite,
     Version = ManifestConstants.OrchardCoreVersion,
-    Category = "Deployment"
+    Category = "Deployment",
+    Description = "Provides a way to import and export packages"
```

Comment:
"packages" should be replaced by "recipes".

Reply:
@Skrypt  The menu item is called "Package Import" which is why I referred to it as package. May be the menu should be also renamed. May be even combine "JSON Import" and Recipe Import into a single page with option to upload a file of just provide a string.
Any way, the request change is done.